nais/dev/mock.json

```diff
@@ -2,6 +2,7 @@
   "applicationName": "sosialhjelp-mock-alt-api-mock",
   "namespace": "teamdigisos",
   "ingress": [
+    "https://sosialhjelp-mock-alt-api-mock.ekstern.dev.nav.no/sosialhjelp/mock-alt-api",
```

Comment:
trenger vi denne? 🤔 
trodde `ekstern` var for test-proxyen --> `digisos.ekstern.dev.nav.no`

Reply:
Nei, det gjør vi ikke.